internal/jit/traceplan.go

```
@@ -0,0 +1,453 @@
+			// instead of a deopt round trip. Cuts inside an inlined frame
+			// keep the fallback — the root block expects the anchor frame
+			// only.
+			if rejoins(op) {
+				current.Term = Terminator{Kind: TerminateBranch, IP: op.Target, Hot: 0, Edges: []Edge{jump(op.Fn, op.Target)}}
+			} else {
+				current.Term = Terminator{Kind: TerminateFallback, IP: op.Target, Hot: -1}
+			}
+			blocks = append(blocks, current)
+			return blocks
+		}
+		path := -1
```

golangci-lint (2.12.2)

[error] ineffectual assignment to path

(ineffassign)